sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/DirectConnectionConfig.java

```diff
@@ -17,24 +17,27 @@
     //  Constants
     private static final Duration DEFAULT_IDLE_ENDPOINT_TIMEOUT = Duration.ofSeconds(70L);
     private static final Duration DEFAULT_CONNECTION_TIMEOUT = Duration.ofSeconds(60L);
+    private static final Duration DEFAULT_REQUEST_TIMEOUT = Duration.ofSeconds(60L);
     private static final int DEFAULT_MAX_CONNECTIONS_PER_ENDPOINT = 30;
     private static final int DEFAULT_MAX_REQUESTS_PER_CONNECTION = 10;
 
     private Duration connectionTimeout;
     private Duration idleConnectionTimeout;
     private Duration idleEndpointTimeout;
+    private Duration requestTimeout;
```

Comment:
Please add this new field to `toString()` API.

Reply:
good catch, thanks